resources/scss/admin-style.scss

```diff
@@ -0,0 +1,303 @@
+@import url('https://fonts.googleapis.com/css2?family=M+PLUS+Rounded+1c:wght@300;400;500;700;800&display=swap');
+
+@import url('https://fonts.googleapis.com/css2?family=Raleway:wght@400;500&display=swap');
+
+/* style.css which kai-san creates  */
+
+:root {
```

Comment:
there's no need for the whole `:root`, you can just replace that with `@import 'variables';`

Reply:
I will replace it.